src/interface.jl

```diff
@@ -338,32 +350,34 @@ function get_sol(mumps::Mumps{T}) where {T}
 end
 
 """
-    get_schur_complement!(S,mumps)
+    get_schur!(S,mumps)
```

Comment:
Let's not change method names in a doc pr.

Reply:
I need this modification to fix the documentation, I opened #97.